aio_pika/robust_connection_multihost.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Optional, Any
+from yarl import URL
+from aio_pika.robust_connection import RobustConnection
+import asyncio
+
+class RobustConnectionMultiHost(RobustConnection):
```

Comment:
`RobustConnectionRRHost` semantically it is closer, because you connection is not a multihost, it is a round-robin retying to connect only to one.